spec/eventq_base/serialization_providers/jruby/oj/serializer_spec.rb

```diff
@@ -48,6 +48,12 @@
       require 'oj'
 
       it 'creates json that CRuby OJ can deserialize' do
+        p '*****************************************'
```

Comment:
This debugging can go right?

Reply:
yes, I have no idea why it is failing on occasion.  Can't get it to fail locally. So when it fails, we can see the info in the Travis logs and figure out what it didn't like.